content/guides/guides/cross-origin-testing.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Cross Origin Testing
+---
+
+<Alert type="info">
+
+<strong class="alert-header"> Note </strong>
+
+As of Cypress [v12.0.0](https://on.cypress.io/changelog#12-0-0), Cypress has the
+capability to visit multiple origins in a single test via the
+[cy.origin()](https://on.cypress.io/origin) command!
+
+</Alert>
+
+Cypress limits each test to visiting domains that share the same superdomain. If
```

Comment:
> Cypress limits each test to visiting domains that share the same superdomain

This statement feels like it counters the above statement

> As of Cypress [v12.0.0](https://on.cypress.io/changelog#12-0-0), Cypress has the
capability to visit multiple origins in a single test 

I assume you're trying to something like: "by default, cypress runs the commands in a test agains a single origin".
(or is is super domain?  this always confuses me since its cy.origin not cy.superdomain)

Reply:
it's confusing, but it is superdomain. Once its outside of same superdomain, you have to use `origin()`, which we want an exact match of the said origin and not the origins superdomain. Not exactly easy to digest, but I think what we have gives a pretty good summary.

Reply:
Without the full context I think it does counter it, but the next sentence ` If
a navigation occurs that does not meet the same superdomain rule, the
[cy.origin](/api/commands/origin) command must be used to execute Cypress commands
inside the newly navigated origin.` I feel like addresses that explanation